_data/toc/digital-workspace.yaml

```diff
@@ -23,6 +23,8 @@
           path: '/digital-workspace/latest/governance/security/'
         - title: 'Working with records'
           path: '/digital-workspace/latest/governance/records/'
+        - title: 'Working with folder rules'
+          path: '/digital-workspace/latest/governance/folder-rules/'
```

Comment:
why the path is governance ? Folder rules is ADW functionality

Reply:
@lilustig Would you prefer the folder rules info in a **Using** section? By doing that we can add more end-user ADW functionality in the same page. I'll leave Oliver to decide on the docs structure.

Reply:
The structure was made a year or so ago before alot of governance items were being added. This structure was agreed with the PM at the time. 

As @anxumalo has said we can make it a Using section which would match the documentation of the other products, now there is more functionality being added.
Can you confirm you would like this doing please? You can see an example of how it can look here: https://docs.alfresco.com/media-management/latest/using/

**Note:** Changing the structure like this has implications for our customers who have already visited the governance section of the ADW docs, and perhaps saved links to it etc.

Reply:
@oliveralfresco2018 Just to clarify ... my proposal was only to move the `folder-rules` page and no other content! Hope that helps 🙂

Reply:
Ive created a Using section now.